indico/modules/events/registration/fields/base.py

```diff
@@ -94,6 +97,19 @@ def create_setup_schema(self):
         schema = self.setup_schema_base_cls.from_dict(self.setup_schema_fields, name=name)
         return schema()
 
+    def create_mm_field(self, registration):
```

Comment:
default `registration` to `None`? (seems logical even if we always pass it in the end)